src/components/home/Home.jsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import { Header } from "./sections/Header";
+
+export const Home = () => {
+  return (
+    <div>
+      <Header />
+      <br />
```

Comment:
let's delete these 3 BRs here, generally we don't use BR like this. if you want to add space use a bottom margin on header